sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -171,17 +171,19 @@ private[sql] class SharedState(
 
 object SharedState extends Logging {
   @volatile private var fsUrlStreamHandlerFactoryInitialized = false
-  private var sharedState: SharedState = _
+  @volatile private var _sharedState: SharedState = _
 
-  def getSharedState(
-      sparkContext: SparkContext,
-      initialConfigs: scala.collection.Map[String, String])
-    : SharedState = synchronized {
-    if (sharedState == null) {
-      // Ensure that we only initiate SharedState once
-      sharedState = new SharedState(sparkContext, initialConfigs)
+  def get(sparkContext: SparkContext, initialConfigs: scala.collection.Map[String, String])
+    : SharedState = {
+    if (_sharedState == null) {
+      synchronized {
+        if (_sharedState == null) {
+          // Ensure that we only initiate SharedState once
```

Comment:
nit: Remove it or (rather not) make it part of scaladoc.

Reply:
removed. thanks!